LenovoLegionToolkit.Lib/Controllers/DisplayBrightnessController.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace LenovoLegionToolkit.Lib.Controllers
+{
+    public class DisplayBrightnessController
+    {
+        public Task<int> Get()
+        {
+            using var mclass = new ManagementClass("WmiMonitorBrightness")
+            {
+                Scope = new ManagementScope(@"\\.\root\wmi")
+            };
+            using var instances = mclass.GetInstances();
+            foreach (ManagementObject instance in instances)
+            {
+                return Task.FromResult((int)(byte)instance.GetPropertyValue("CurrentBrightness"));
+            }
+            return Task.FromResult(0);
```

Comment:
It would be a better idea to throw an exception here if data is not found.

Reply:
Added your `WMI` class. This should take care of exceptions.